subgraphs/_reference_/src/sdk/protocols/generic/index.ts

```diff
@@ -0,0 +1,70 @@
+import { ethereum } from "@graphprotocol/graph-ts";
+
+import { AccountManager } from "./account";
+import { ProtocolManager } from "./protocol";
+import { PoolManager } from "./pool";
+import { TokenPricer } from "../config";
+import { Config } from "./config";
```

Comment:
And then here use the interface in that same `/sdk/protocols/config.ts`: `ProtocolConfigurer`

Reply:
Use the interface or the class?

Reply:
Got it. Updated!